kasmvnc/run.sh

```diff
@@ -159,21 +154,29 @@ else
 fi
 
 # Coder port-forwarding from dashboard only supports HTTP
-sudo bash -c "cat > /etc/kasmvnc/kasmvnc.yaml <<EOF
+sudo tee /etc/kasmvnc/kasmvnc.yaml > /dev/null << EOF
 network:
   protocol: http
   websocket_port: ${PORT}
   ssl:
     require_ssl: false
   udp:
     public_ip: 127.0.0.1
-EOF"
+EOF
```

Comment:
For images without `sudo`
we can also write the same file in `$HOME/.vnc/kasmvnc.yaml`, and it overrides the one in `/etc/kasmvnc/kasmvnc.yaml` if one exists.

Reply:
Should we just default to using the home directory?

Reply:
Probably yes. But for the official kasm images, I have observed they bake a kasm.yaml that requires SSL certs, so I had to extend/patch the image.

Reply:
I saw that issue after posting this PR. I tried to run postman with these changes, but the workspace never finished starting up.
But, if we write our version to the home directory, then it should always use that one and shouldn't matter what they baked in?

Reply:
What if we change it to:
```yaml
network:
  protocol: http
  websocket_port: ${PORT}
  ssl:
    require_ssl: false
    pem_certificate:
    pem_key:
  udp:
    public_ip: 127.0.0.1
```

Reply:
This works 🎉 . I thought blank would be an invalid value. But it looks like it's still a valid string.